opds_import.py

```diff
@@ -250,14 +363,14 @@ def add_with_metadata(self, feed):
         add_with_metadata_url = self.get_collection_url(self.ADD_WITH_METADATA_ENDPOINT)
         return self._post(add_with_metadata_url, unicode(feed))
 
-    def metadata_needed(self):
+    def metadata_needed(self, **kwargs):
```

Comment:
These methods need to start taking **kwargs because they're being called with the `allowed_response_codes` keyword argument.